inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/pad.cpp

```diff
@@ -19,16 +19,60 @@ const std::vector<InferenceEngine::Precision> netPrecisions = {
         InferenceEngine::Precision::U8,
 };
 
-const std::vector<std::vector<int64_t>> padsBegin2D = {{0, 0}, {1, 1}, {2, 0}, {0, 3}};
-const std::vector<std::vector<int64_t>> padsEnd2D   = {{0, 0}, {1, 1}, {0, 1}, {3, 2}};
-const std::vector<float> argPadValue = {0.f, 1.f, -1.f, 2.5f};
+const std::vector<float> argPadValue = {{}, 1.f, -1.f, 2.5f};
```

Comment:
M.b. I do not know something, but what is the point of using default initialization `{}` of a `float`?  It will be zero-initialized just as it was before.

Reply:
It was check of my guess. Returned last behavior. Thanks!